### PR TITLE
Add extraEnvFromSecrets value to db chart

### PR DESCRIPTION
**Pull request by shepz**
### Description

This PR introduces a new Helm chart value: `extraEnvFromSecrets`. This addition is beneficial for setups where secrets are managed by an external secret manager and synced through an external process.

### CODE REVIEW

**Comment by shepz** [comment]
@faridco I can’t assign reviewers, could you take a look at this PR?

File: aidbox/values.yaml [hunk 1]
```diff
@@ -10,6 +10,9 @@ config:
   BOX_METRICS_PORT: 8765
   PGPORT: 5432
 
+auth:
+  existingSecret: ""
```
Comment (faridco) [inline aidbox/values.yaml:14]: there is `.Values.extraEnvFromSecrets` and it should cover external like secrets

similar approach could be applied to the AidboxDB

Comment (shepz) [reply aidbox/values.yaml:14]: Sure, I can update the PR to only add `extraEnvFromSecrets` to the db chart if that’s what you prefer. My concern is that we’re still creating an empty secret, which could add confusion when verifying which secrets are in use. That’s why I leveraged the `auth.existingSecret` logic commonly found in mainstream Helm charts (like Bitnami). But I’ll go with your call!

**Review by faridco — APPROVED** [review]
thank you

**Comment by faridco** [comment]
released new AidboxDB chart version

**Pull request merged**